.rspec

```diff
@@ -1,3 +1,2 @@
 --warnings
 --color
---require spec_helper
```

Comment:
Please add this back, it's our project standard to require this here and not in spec files.